app/jobs/resolution_proofing_job.rb

```diff
@@ -76,7 +76,7 @@ def perform(
 
     if IdentityConfig.store.idv_socure_shadow_mode_enabled
```

Comment:
issue (blocking): This should also respect the `socure_enabled` config flag so that this will not run when executed out-of-band.

Reply:
this fag can be renamed ... it's use is inteded only for DocV

rename to `socure_docv_enabled` ?